pkg/s3/client.go

```diff
@@ -141,17 +155,75 @@ func (client *s3Client) removeObjects(bucketName, prefix string) error {
 			GovernanceBypass: true,
 		}
 		errorCh := client.minio.RemoveObjects(client.ctx, bucketName, objectsCh, opts)
+		haveErrWhenRemoveObjects := false
 		for e := range errorCh {
-			glog.Errorf("Failed to remove object %s, error: %s", e.ObjectName, e.Err)
+			if e.Err.Error() != "EOF" {
```

Comment:
Will this ever be `"EOF"`? I could not find anything in the minio client about that.

Reply:
```
I0408 14:22:31.691835       1 utils.go:97] GRPC call: /csi.v1.Controller/DeleteVolume
I0408 14:22:31.691860       1 controllerserver.go:152] Deleting volume boxjan-k8s-storage/pvc-e1bc1d78-43b4-4b8d-9133-f5ccf69805f9
E0408 14:22:32.008419       1 client.go:157] Failed to remove object @, error: Server does not support one or more requested headers. Please see https://developers.digitalocean.com/documentation/spaces/#aws-s3-compatibility
E0408 14:22:32.008572       1 client.go:157] Failed to remove object @, error: EOF
W0408 14:22:32.008659       1 client.go:103] removeObjects failed with: Failed to remove all objects of bucket boxjan-k8s-storage, will try removeObjectsOneByOne
I0408 14:22:32.122071       1 controllerserver.go:179] Bucket boxjan-k8s-storage/pvc-e1bc1d78-43b4-4b8d-9133-f5ccf69805f9 is not created by csi-s3, will not be deleted by csi-s3 automatically.
```
log format is `glog.Errorf("Failed to remove object %s@%s, error: %s", e.ObjectName, e.VersionID, e.Err)`
I got these log in my cluster, when use RemoveObjects, I'm not sure where it from, api or minio.

Reply:
Hm not sure but I assume these are coming from DigitalOcean API. Anyway, even if it comes from minio, if it returns the error with the string "EOF" I would prefer to know about it and not ignore it. In this case `haveErrWhenRemoveObjects` would have been true anyway as there was another error just before that one. So I would suggest to just revert to the simple `if len(errorCh) != 0 {` to catch remove errors.

Reply:
No, If you use range to get all err in channel, after broken the loop, the length of errorCh always 0.

Reply:
Oh, you are right. But can you still remove the skipping these EOF errors? Other than that I think we can merge this.

Reply:
ok, I will push later.

Reply:
I've changed it.